src/main/java/rs/teslaris/core/service/impl/document/JournalPublicationServiceImpl.java

```diff
@@ -34,19 +35,19 @@ public class JournalPublicationServiceImpl extends DocumentPublicationServiceImp
     private final DocumentPublicationIndexRepository documentPublicationIndexRepository;
 
     @Autowired
-    public JournalPublicationServiceImpl(
-        MultilingualContentService multilingualContentService,
-        DocumentPublicationIndexRepository documentPublicationIndexRepository,
-        DocumentRepository documentRepository,
-        DocumentFileService documentFileService,
-        PersonContributionService personContributionService,
-        SearchService<DocumentPublicationIndex> searchService,
-        JournalPublicationJPAServiceImpl journalPublicationJPAService,
-        JournalService journalService,
-        JournalPublicationRepository journalPublicationRepository,
-        DocumentPublicationIndexRepository documentPublicationIndexRepository1) {
+    public JournalPublicationServiceImpl(MultilingualContentService multilingualContentService,
```

Comment:
do we need one class for two subclasses of PublicationSeries (Journal and BookSeries) or two separaty service implementations? Anyway it is not well implemented at the moment.

Reply:
This is a topic for discussion, I advocate two separate service classes because that way we can ensure that Journal and BookSeries model classes can evolve separately (even though they are identical at this moment) and avoid close coupling of objects. If we are absolutely sure that Journal and BookSeries model stays the same, then one class should be ok.

Reply:
I fully agree, and I think we should start with two separate services (for Journal and BookSeries). Please add that.